docs/src/embeddings/default_embedding_functions.md

```diff
@@ -186,21 +186,26 @@ We offer support for all huggingface models (which can be loaded via [transforme
 Example usage - 
 ```python
 import lancedb
-import pandas as pd
-
 from lancedb.embeddings import get_registry
 from lancedb.pydantic import LanceModel, Vector
 
+db = lancedb.connect("/tmp/db")
 model = get_registry().get("huggingface").create(name='facebook/bart-base')
 
-class TextModel(LanceModel):
+
+class Words(LanceModel):
```

Comment:
Can we keep TextModel ?
the name is more apt than 'Words'

Reply:
Thanks for the comment, happy to use this - would you like me to change all the other examples to match this, i.e.:

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L164

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L226

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L280

Reply:
Please change only where `Words` is the class name, Schema is ok, Thank you